integration-tests/js-compute/package.json

```diff
@@ -14,5 +14,8 @@
   },
   "dependencies": {
     "@fastly/js-compute": "file:../../sdk/js-compute"
+  },
+  "volta": {
+    "node": "16.15.1"
```

Comment:
This is useful to me as I use https://volta.sh/ to pick which version of NodeJS and/or npm that each folder/project should use.

Reply:
Should we standardize on using this for development?